source/_components/volvooncall.markdown

```diff
@@ -0,0 +1,39 @@
+---
+layout: page
+title: "Volvo On Call"
+description: "Instructions for how to integrate Volvo On Call into Home Assistant."
+date: 2016-10-02 17:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: volvo.png
+ha_category: Hub
+ha_release: 0.39
+---
+
+
+The `volvooncall` platform offers integrates with the [Volvo On Call](http://www.volvocars.com/intl/own/connectivity/volvo-on-call) cloud service and offers presence detection as well as sensors such as odometer and fuel level.
+
+To use Volvo On Call in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
```

Comment:
Looks good but I suggest you have a minimal example and this one as a advanced example.